archinstall/lib/installer.py

```diff
@@ -667,7 +667,7 @@ def enable_service(self, services: str | list[str]) -> None:
 			info(f'Enabling service {service}')
 
 			try:
-				self.arch_chroot(f'systemctl enable {service}')
+				SysCommand(f'systemctl --root={self.target} enable {service}')
```

Comment:
This operates on a chroot in a way native to systemd tools and should be done regardless of arch-chroot working or not.

Reply:
We have to remember that we're on Arch Linux and the recommended way *(by our own Wiki)* is to configure the system using `arch-chroot`, generally speaking.

It's unfortunate that this issue happens, and we'll try to work around it.
But we wish to use native tooling with default values wherever possible and not use workarounds or too many custom parameters *(again, if possible)*.